import_export/fields.py

```diff
@@ -123,3 +126,15 @@ def export(self, obj):
         if value is None:
             return ""
         return self.widget.render(value, obj)
+
+    def _get_dehydrate_method(self, field_name=None):
```

Comment:
I suggest to remove the leading underscore and call the method `get_dehydrate_method()`.  I believe that since this method is called from outside the class, the convention would be not to use a leading underscore ([source](https://stackoverflow.com/a/1301369/39296))